bin/cylc-show

```diff
@@ -46,6 +48,10 @@ def main():
     parser.add_option('--list-prereqs', action="store_true", default=False,
                       help="Print a task's pre-requisites as a list.")
 
+    parser.add_option('--json', action="store_true", default=False,
+                      help="Print all tasks and corresponding requisites in "
+                           "JSON format.")
+
```

Comment:
I think it would be sufficient to say "Print output in JSON format" here.

Reply:
Good suggestions @hjoliver, I will get them implemented on Monday.